app.js

```diff
@@ -0,0 +1,140 @@
+const express = require("express")
+const cors = require("cors")
+const bodyParser = require("body-parser")
+const time = require("express-timestamp")
+
+
+const port = 3000
+
+const classRouter = require('./Class.js')
+const StudentRouter = require('./Student.js')
+const SchoolRouter = require("./School.js")
+
+const app = express()
+app.use(time.init);
+
+app.use(cors())
+app.use(bodyParser.urlencoded({
+    extended: false
+}))
+app.use(bodyParser.json())
+
+const School = require("./School.js")
+const Student = require("./Student.js")
+// const Classes = require("./Class.js")
+
+let mySchool = new School();
+
+
+app.get("/", (req, res) => {
+    res.json(newSchool);
+});
+
+
+// adding a class 
+
+// app.post("/class", (req, res) => {
+//             if (!mySchool.addClass(req.body.name, req.body.teacher)) {
+//                 res.json({
+//                     "error": "Please fill out all the information or Class already exists",
+//                     "timestamp": req.timestamp
+//                 })
+//             } else {
+//                 res.json({
+//                         "class": {
+//                             "name": req.body.name,
+//                             "teacher": req.body.teacher,
+//                             "students": []
+            
+//                         })
+//                 )
+//         })
+
+            app.post("/class", (req, res) => {
+                try {
+                    // let addNewClass = mySchool.addClass
+                    mySchool.addClass(req.body.name, req.body.teacher);
+                    // if(newSchool.classes === req.body.name || req.body.teacher)
+                    res.json({
+                        "class": {
+                            "name": req.body.name,
```

Comment:
You can't send your request back to the frontend inside of your response. Your response should come 100% from your backend. You can assign the name to `newClass.name` and teacher to `newClass.teacher` . Or you can simply assign the key of `class` to the value of `newClass` and parse through it on the frontend.